tools/PI/DevHome.PI/Helpers/ExternalTool_v1.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json.Serialization;
+using CommunityToolkit.Mvvm.ComponentModel;
+
+namespace DevHome.PI.Helpers;
+
+public enum ExternalToolArgType
+{
+    None,
+    ProcessId,
+    Hwnd,
+}
+
+public partial class ExternalTool_v1 : ObservableObject
```

Comment:
v1?  Why v1?  Is this class definition not allowed to change?

Reply:
Correct: this is the legacy type definition, and it is not allowed to change.